application_form/api/sales/serializers.py

```diff
@@ -226,7 +226,7 @@ def validate_apartment_uuid(self, value: UUID) -> UUID:
         return value
 
     def create(self, validated_data):
-        return create_reservation_without_application(
+        return create_late_application(
```

Comment:
It's creating a reservation, not application, so maybe `create_late_reservation`